service/dap/server.go

```diff
@@ -1832,7 +1832,10 @@ type convertVariableFlags uint8
 
 const (
 	skipRef convertVariableFlags = 1 << iota
-	showFullValue
+	// A variable is considered basic if it cannot be expanded to get additional
+	// values.
+	showFullValueBasic
+	showFullValueAll
```

Comment:
how about:
showFullValue and showFullValueIfNoChildren

Reply:
Not applicable anymore.